scripts/fetch.sh

```diff
@@ -228,7 +228,8 @@ sync_references releases grpc-ecosystem grpc-gateway https://github.com/grpc-eco
 sync_references releases opencensus opencensus https://github.com/census-instrumentation/opencensus-proto src
 sync_references releases opentelemetry opentelemetry https://github.com/open-telemetry/opentelemetry-proto
 sync_references releases prometheus client-model https://github.com/prometheus/client_model
-sync_references releases protocolbuffers gofeatures https://github.com/protocolbuffers/protobuf-go src
+# TODO: remove after it finishes syncing its README's deprecation notice in existing BSRs.
+# sync_references releases protocolbuffers gofeatures https://github.com/protocolbuffers/protobuf-go src
```

Comment:
Should we remove the CODEOWNERS entry now too?

Reply:
No need yet, we want to be aware of any changes, although there shouldn't be any.